tests/system/test_integration.py

```diff
@@ -40,14 +40,16 @@ def test_load_docs_with_template_and_add_transaction(self):
         rs = self.es.search(index=self.index_name, body={
             "query": {"term": {"processor.event": "transaction"}}})
         assert rs['hits']['total'] == 4, "found {} documents".format(rs['count'])
-        approved = json.load(open('transaction.approved.json'))
+        with open(self._beat_path_join(os.path.dirname(__file__), 'transaction.approved.json')) as f:
```

Comment:
not introduced with this PR, but still: I would recommend to use [`codecs.open(..., encoding='utf-8')`](https://docs.python.org/3/library/codecs.html#codecs.open) with content that may very well contain non-ASCII characters

Reply:
good call, this actually bit us recently elsewhere - I'll make an issue to address both.